### PR TITLE
307 kill seq tests

### PR DESCRIPTION
**Pull request by n1v0lg**
- Removes sequential tests from spdz/ dummy arithmetic suites
- Uses smaller modulus where possible for spdz
- Sets up AbstractSpdzTest for more refactoring

### CODE REVIEW

File: core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/AbstractDummyArithmeticTest.java [hunk 1]
```diff
@@ -77,6 +77,9 @@ private void runTest(
       TestThreadRunner.TestThreadFactory<DummyArithmeticResourcePool, ProtocolBuilderNumeric> f,
       EvaluationStrategy evalStrategy, int noOfParties, BigInteger mod, int maxBitLength,
       int fixedPointPrecision, boolean logPerformance) {
+    if (evalStrategy == EvaluationStrategy.SEQUENTIAL) {
+      throw new IllegalArgumentException(":(");
```
Comment (GuutBoy) [inline core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/AbstractDummyArithmeticTest.java]: Maybe pass a more useful string argument to this exception?

Comment (GuutBoy) [reply core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/AbstractDummyArithmeticTest.java]: Actually, maybe not throw an exception?

File: suite/spdz/src/test/java/dk/alexandra/fresco/suite/spdz/AbstractSpdzTest.java [hunk 2]
```diff
@@ -64,23 +64,43 @@
  */
 public abstract class AbstractSpdzTest {
 
-  protected Map<Integer, PerformanceLogger> performanceLoggers = new HashMap<>();
+  private Map<Integer, PerformanceLogger> performanceLoggers = new HashMap<>();
   // TODO hack hack hack
-  private static final int DEFAULT_MOD_BIT_LENGTH = 512;
-  private static final int DEFAULT_MAX_BIT_LENGTH = 150;
+//  private static final int DEFAULT_MOD_BIT_LENGTH = 512;
+//  private static final int DEFAULT_MAX_BIT_LENGTH = 150;
```
Comment (GuutBoy) [inline suite/spdz/src/test/java/dk/alexandra/fresco/suite/spdz/AbstractSpdzTest.java]: Delete these comments.

File: core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/AbstractDummyArithmeticTest.java (hunk 1 above)
Comment (n1v0lg) [reply core/src/test/java/dk/alexandra/fresco/suite/dummy/arithmetic/AbstractDummyArithmeticTest.java]: Ah, good catch, forgot to delete that.

**Pull request merged**